docs/advanced/verifying-binaries.md

```diff
@@ -1,53 +1,305 @@
+relies on hashing. The Decred team takes it a step further and uses
+[GnuPG](https://gnupg.org/index.html) to sign their hashes so you can verify
+that the file came directly from the Decred team.
+
+When the Decred team [compiles source code into binaries](https://github.com/decred/decred-release),
+they hash each binary using [SHA-256](https://en.wikipedia.org/wiki/SHA-2).
+Each hash is stored in a `manifest.txt` file. Then they sign the manifest using
+their public GnuPG key. This creates a signature file, ending in
+`manifest.txt.asc`.
+
+To check if your downloaded binary is a 100% match to the official Decred
+binary, you need to do two things:
+
+1. **Verify that the manifest signature was directly signed by Decred:**
+   Download the `manifest.txt` and the signature `manifest.txt.asc`. Use GnuPG
+   to check the signature against the public Decred Release Signing Key.
+
+     * SUCCESS: If the signatures match, you know you can trust that the
+       manifest came directly from Decred, and that the hashes inside it are
+       trustworthy.
+
+     * FAIL: If the signatures don't match, then your manifest is not the
+       official Decred manifest. Do not continue, delete the files you've
+       downloaded, and check for security problems before trying again.
+
+2. **Verify that the binary hash matches the manifest hash:** Hash your
+   downloaded binary and verify that its hash matches the hash in the official
+   Decred `manifest.txt`.
+
+     * SUCCESS: If your hash matches the hash in the manifest, you know you can
+       trust and run the binary.
+
+     * FAIL: If your hash doesn't match, your binary doesn't match the official
+       Decred binary release, and you should not continue. Delete the binary
+       you've downloaded, and check for security problems before trying again.
+
+## Detailed Instructions
+
+### Preliminary Steps
+
+First, go to the official [Decred releases](https://github.com/decred/decred-release/releases)
+page and download:
+
+* The binary installer for your OS / architecture
+* The file manifest and hashes, ending in `-manifest.txt`
+* The signature for the manifest, ending in `-manifest.txt.asc`
+
+Also, make sure GnuPG is installed:
+
+* On Windows, install GnuPG with the latest version of [gpg4win](https://www.gpg4win.org/).
+* On macOS, install GnuPG with the latest version of [GPGTools](https://gpgtools.org/).
+* On Linux, GnuPG is installed by default.
+
+With GnuPG installed, the following instructions will work from any terminal on
+Windows, macOS, or Linux/UNIX
+
+### Add the official Decred Release public keys to your GnuPG keyring.
+
+You only need to do this step one time, so you can skip this step when
+verifying later releases on the same computer.
+
+There are several ways to import the Decred Release public keys into GnuPG.
+The most direct method is to use the ASCII Key Block below:
+
+<details><summary><b>Here's Decred's ASCII Key Block!</b></summary>
```

Comment:
Hmmm...I tried this with `??? info "title"`, and indenting the entire Key Block by 4 spaces, but the code block no longer works.
....\```
....key block
....\```

I tried it with just a test sample:
```
??? info "test"

    ```
    line 1
    line 2
    ```
```
 
And the output is just `line 1 line 2`.

If you don't mind, I'd like to keep the inline HTML.

Reply:
The reason I want to use a code block is so that markdown allows instantly copying the whole thing.

Reply:
In this case the indenting will require 8 spaces. 4 to indicate a code block, and 4 to indicate that the code block is inside the admonition.

```
??? info "Here's Decred's ASCII Key Block!"

        -----BEGIN PGP PUBLIC KEY BLOCK-----
        
        mQINBFapILEBEADZxw+4Z8LlqsXCz3j3Ap04SF8zYenlsw123OJZEh9RFERd19bo
        +l2RueFqi5vJDGWpXZ+eHxvgevvOO3r0AiIgAByAP7RQQxip4j6M2xnEBdVb9UV5
        baO93JcyBRDnII/zh6Zf4pqngiYEz7juySsnVMrE7IFmIdT/WfoGW6FX8/kRXyzf
```

Markdown only recognises fences (triple backticks) when they are the first character on a line, otherwise you need to use indentation

Reply:
Perfect. Fixed.